test/image/compare_pixels_test.js

```diff
@@ -93,7 +93,8 @@ if(allMock || argv.filter) {
     allMockList = allMockList.filter(function(mockName) {
         var cond = !(
             mockName === 'font-wishlist' ||
-            mockName.indexOf('mapbox_') !== -1
+            mockName.indexOf('mapbox_') !== -1 ||
+            mockName.match(/image_.*source/)
```

Comment:
Making the baseline via `npm run baseline` didn't work for some reason. I wonder if it has to do with the fact that the `plot` step is now async.

Reply:
Even after e4b923b9c043f71f98dde8de38bd830f96db34b5, generate the baseline via `npm run baseline` still fails :confused:

Reply:
I don't know if this may help you debug; the error is 
```
{ [Error: socket hang up] code: 'ECONNRESET' }
```

Reply:
Yeah... It doesn't tell much. However, it does work in Orca so I can't justify looking into the old image renderer.

Reply:
Hopefully, it'll work in Kaleido and we'll start using it in our `test-image`.